README.md

```diff
@@ -5,22 +5,61 @@
 <!-- default badges end -->
 # Data Grid for WPF - How to Specify Edit Form Settings
 
-<!-- default file list -->
+This example illustrates how to specify Edit Form settings when a user starts to edit a row. You can initialize values or make certain editors read-only. To do that, handle the [RowEditStarting](https://docs.devexpress.com/WPF/DevExpress.Xpf.Grid.TableView.RowEditStarting) / [NodeEditStarting](https://docs.devexpress.com/WPF/DevExpress.Xpf.Grid.TreeListView.NodeEditStarting) event or create a command in a View Model and bind it to the [RowEditStartingCommand](https://docs.devexpress.com/WPF/DevExpress.Xpf.Grid.TableView.RowEditStartingCommand) / [NodeEditStartingCommand](https://docs.devexpress.com/WPF/DevExpress.Xpf.Grid.TreeListView.NodeEditStarting) property. The event / command arguments contain the **CellEditors** property that returns an array of [CellEditorData](https://docs.devexpress.com/CoreLibraries/DevExpress.Mvvm.CellEditorData) objects. Each object allows you to specify the corresponding editor's settings.
```

Comment:
Does "To do that" refer to preceding sentence ("initialize values or make certain editors read-only") or the initial sentence ("specify Edit Form settings")? Can you make it more specific so that the reader will know?

For example, if "To do that" refers to the preceding sentence, then we could say something like "To initialize values or make certain editors read-only, ..."
Let me know what you think.

Reply:
Yes, "To do that" refers to the preceding sentence  ("initialize values or make certain editors read-only") . I made changes according your suggestion. Thank you.

Reply:
Thank you!